benchmarks/samples/fabric/fabcar/queryCar.js

```diff
@@ -0,0 +1,61 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+module.exports.info = 'Querying a car.';
+
+let txIndex = 0;
+let carIndex = 0;
+let carNumber;
+let bc, contx;
+
+module.exports.init = async function(blockchain, context, args) {
+    bc = blockchain;
+    contx = context;
+
+    while (carIndex < args.assets) {
+        carIndex++;
+
+        carNumber = carIndex.toString();
```

Comment:
This car creation stage is pretty handy and can be reused in other benchmarks - breaking it out into a utility function would make sense here 👍 then u can reuse it in all query style tests to populate test data